addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/NetatmoBindingConstants.java

```diff
@@ -52,10 +52,10 @@
     public static final String CHANNEL_LOCATION = "Location";
     public static final String CHANNEL_BOILER_ON = "BoilerOn";
     public static final String CHANNEL_BOILER_OFF = "BoilerOff";
-    public static final String CHANNEL_DATE_MAX_TEMP = "date_max_temp";
-    public static final String CHANNEL_DATE_MIN_TEMP = "date_min_temp";
-    public static final String CHANNEL_MAX_TEMP = "min_temp";
-    public static final String CHANNEL_MIN_TEMP = "max_temp";
+    public static final String CHANNEL_DATE_MAX_TEMP = "DateMaxTemp";
+    public static final String CHANNEL_DATE_MIN_TEMP = "DateMinTemp";
+    public static final String CHANNEL_MAX_TEMP = "MaxTemp";
+    public static final String CHANNEL_MIN_TEMP = "MinTemp";
```

Comment:
@clinique @lolodomo Another question should they be added somewhere, for example to the `MEASURABLE_CHANNELS`